frontend/server/src/main/java/org/pytorch/serve/wlm/ModelManager.java

```diff
@@ -152,6 +158,27 @@ private ModelArchive createModelArchive(
         return archive;
     }
 
+    private void setupModelDependencies(Model model) throws IOException, InterruptedException {
```

Comment:
How about supporting custom package given as tar/zip or some dist file?

Reply:
This is already supported. Updated the documentation accordingly.